### PR TITLE
Adds issues workflow

### PR DESCRIPTION
**Pull request by saltiyazan**
# Description

Uses issues workflow to sync issues to jira.

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [x] I validated that new and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] I have bumped the version of the library

### CODE REVIEW

**Review by Gmerold — APPROVED** [review]

File: tests/unit/test_charm.py [hunk 1]
```diff
@@ -301,7 +301,7 @@ def test_relations_available_and_config_pushed_and_pebble_layer_already_applied_
             mounts={
                 "config_dir": Mount(
                     "/free5gc/config/nssfcfg.conf",
-                    Path(__file__).parent / "expected_config" / "config.conf",
+                    Path(__file__).parent / "actual_config" / "config.conf",
```
Comment (Gmerold) [inline tests/unit/test_charm.py:304]: After adding `.strip()` I don't think we need this file anymore. I checked on both my laptop and GH and it works fine.

File: src/charm.py [hunk 2]
```diff
@@ -332,7 +332,7 @@ def _config_file_content_matches(self, content: str) -> bool:
         f"{CONFIG_DIR}/{CONFIG_FILE_NAME}"
```
Comment (Gmerold) [inline src/charm.py:332]: Not touched by you, but it looks like we don't need it here ;)

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-ops==2.4.1
+ops
```
Comment (Gmerold) [inline requirements.txt:1]: I think all changes related to unpinning the `ops` version deserve a separate PR.

**Review by Gmerold — CHANGES_REQUESTED** [review]

File: tests/unit/test_charm.py (hunk 1 above)
Comment (saltiyazan) [reply tests/unit/test_charm.py:304]: It won't fail, at least for now, but this test expects that config not to change, while it is written by other tests, this means that changing a test case might cause this test to fail without any change in the logic.

File: requirements.txt (hunk 3 above)
Comment (saltiyazan) [reply requirements.txt:1]: https://github.com/canonical/sdcore-nssf-operator/pull/20

PR created for that.
Once merged, I'll rebase this one.

Comment (saltiyazan) [reply requirements.txt]: Rebased, now only the issues workflow is contained in this pr.

**Review by Gmerold — APPROVED** [review]
